Project.toml

```diff
@@ -4,6 +4,7 @@ version = "0.11.3"
 
 [deps]
 Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
```

Comment:
We should put the DataFrames-specific code behind an `@requires` block in `__init__()` so that we don't take on a DataFrames dependency. We should only need the Tables direct dependency.

Reply:
I've put it in the @requires block, but then the exported functions and structures are not available after importing the package into env.  I'll check out later how to handle this